src/nmodl/lex.l

```diff
@@ -98,6 +98,14 @@ ELSE	{ /* translate to lower case */
 	return s->type;
 }
 
+[a-zA-Z][a-zA-Z0-9_]*:[a-zA-Z0-9_]* { /*ONTOLOGY_ID*/
```

Comment:
just to clarify : this also needs to be start with `alphabet`?

Reply:
I just checked EBI's onotology list, and every ontology short name starts with a letter (actually, they consist entirely of letters): https://www.ebi.ac.uk/ols/ontologies

Reply:
I thought of two other things about this tokenization:

We should probably allow the optional use of Safe CURIES, e.g. [SAO:830368389] in addition to just SAO:830368389.

Does the current version of nrnivmodl allow a comment colon to begin immediately after a token, and if so does this cause problems with that?

Reply:
Update: NEURON currently does allow comments to start immediately after a token, so

`USEION na READ ena WRITE ina:REPRESENTS CHEBI:29101`

is valid in 7.6, but raises an error in the pull request, because it interprets ina:REPRESENTS as an ONTOLOGY_ID not a token and a comment.

Reply:
The above issue has been fixed.